compiler/core/lam_util.ml

```diff
@@ -45,6 +45,9 @@ let refine_let ~kind param (arg : Lam.t) (l : Lam.t) : Lam.t =
     | Lam_primitive.Pmakeblock _ -> true
     | _ -> false
   in
+  (* Helper for spotting RHS expressions that are safe to inline everywhere.
+     These are the obviously pure shapes where substituting the original value
+     cannot introduce extra work or side effects. *)
```

Comment:
I don't understand. "does not have side effects" is enough of a description? If so isn't there a function already.
Otherwise, let's keep it as is.

Reply:
Oh no whitespace changes all over the file, one moment